projects/import-gitlab-client/tsconfig.lib.prod.json

```diff
@@ -1,5 +1,8 @@
 {
   "extends": "./tsconfig.lib.json",
+  "compilerOptions": {
+    "declarationMap": false
+  },
   "angularCompilerOptions": {
     "enableIvy": false
   }
```

Comment:
We should have a goal to re-enable these imo.

Reply:
It is enabled in the root tsconfig. But yes, they should be enabled in the project APIs as well. I'll create a new draft PR that fixes this.

Reply:
I attempted it but failed: #30 

One thing I learned from digging through docs is that only the root project needs to have ivy enabled since Angular has its `ngcc`. See here:

> Ivy applications can still consume the View Engine format because the Angular compatibility compiler, or ngcc, can convert it to Ivy.
>
> https://angular.io/guide/creating-libraries#transitioning-libraries-to-partial-ivy-format